### PR TITLE
Create CSS Entry for grid

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Create entry for docs/content/css/concepts/grids/terms/grid

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by SSwiniarski** [comment]
Orphan entry from original assignment sheet

File: content/css/concepts/grids/terms/grid/grid.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+grid: <grid-auto-rows> / <grid-template-columns>;
+
+grid: <grid-template-rows> / <grid-auto-flow> <grid-auto-columns>;
+
+grid: <grid-auto-flow> <grid-auto-rows> / <grid-template-columns>;
+```
+
+## Example 1
+
```
Comment (yangc95) [inline content/css/concepts/grids/terms/grid/grid.md:42]:
```suggestion

Define a grid with three rows that are 200-pixel high, and two columns; the first column at `25%` wide, and the second at 75% wide.

```

File: content/css/concepts/grids/terms/grid/grid.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```css
+.my-grid {
+  display: grid;
+  grid: repeat(3, 200px) / 25% 75%;
+}
+```
+
+Defines a grid that has first three rows 200px tall, one column at 25% and the other column at 75%.
+
```
Comment (yangc95) [inline content/css/concepts/grids/terms/grid/grid.md:51]:
```suggestion
```

**Review by yangc95 — COMMENTED** [review]
@SSwiniarski done with first review 👍

**Comment by SSwiniarski** [comment]
@yangc95, I implemented your suggestions.

**Review by yangc95 — APPROVED** [review]
Ready for second review 👍

File: content/css/concepts/grids/terms/grid/grid.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+grid: <grid-auto-rows> / <grid-template-columns>;
+
+grid: <grid-template-rows> / <grid-auto-flow> <grid-auto-columns>;
+
+grid: <grid-auto-flow> <grid-auto-rows> / <grid-template-columns>;
+```
+
+## Example 1
+
+Define a grid with three rows that are 200-pixel high, and two columns; the first column at `25%` wide, and the second at 75% wide.
```
Comment (Dusch4593) [inline content/css/concepts/grids/terms/grid/grid.md:43]:
```suggestion
In the example below is a `grid` declaration for the rows and column (separated by a `/`) with the following:

* There are three rows that are 200-pixels high each.
* There are two columns, with one being `25%` of the grid's width and another at `75%`-wide.
```

File: content/css/concepts/grids/terms/grid/grid.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+## Example 2
+
+```css
+.my-grid {
+  display: grid;
+  grid: auto / auto auto auto;
+}
+```
+
+Defines a grid that's three columns wide and has auto-sized rows and columns.
```
Comment (Dusch4593) [inline content/css/concepts/grids/terms/grid/grid.md:61]:
```suggestion
Defines a grid that's three columns wide and has auto-sized rows and columns.
```
```suggestion
The example above defines a grid that has three columns. Each column and row is `auto`-sized.
```

File: content/css/concepts/grids/terms/grid/grid.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
+Title: 'grid'
+Description: 'Shorthand for setting grid property values.'
```
Comment (Dusch4593) [inline content/css/concepts/grids/terms/grid/grid.md:3]:
```suggestion
Description: 'Shorthand for setting multiple grid property values.'
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished with my review and left some comments/suggestions for you.

**Comment by SSwiniarski** [comment]
@Dusch4593, I applied your suggestions.

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM 👍

File: content/css/concepts/grids/terms/grid/grid.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+Tags:
+  - 'Grid'
+  - 'Elements'
+  - 'Properties'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `grid` property is shorthand for setting the following properties:
```
Comment (yangc95) [inline content/css/concepts/grids/terms/grid/grid.md:16]:
```suggestion
The `grid` property is a shorthand for setting the following properties:
```

File: content/css/concepts/grids/terms/grid/grid.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+grid: <grid-auto-rows> / <grid-template-columns>;
+
+grid: <grid-template-rows> / <grid-auto-flow> <grid-auto-columns>;
+
+grid: <grid-auto-flow> <grid-auto-rows> / <grid-template-columns>;
+```
+
+## Example 1
+
+In the example below is a `grid` declaration for the rows and column (separated by a `/`) with the following:
```
Comment (yangc95) [inline content/css/concepts/grids/terms/grid/grid.md:43]:
```suggestion
The example below is a `grid` declaration for the rows and columns (separated by a `/`) with the following:
```

File: content/css/concepts/grids/terms/grid/grid.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+- There are three rows that are 200-pixels high each.
```
Comment (yangc95) [inline content/css/concepts/grids/terms/grid/grid.md:45]:
```suggestion
- There are three rows that are each 200-pixels high.
```

**Comment by SSwiniarski** [comment]
@Dusch4593, @yangc95 I committed the last few suggestions.

**Comment by yangc95** [comment]
Entry is live here:

- https://www.codecademy.com/resources/docs/css/grids/grid

✨ Thanks to @SSwiniarski @Dusch4593

**Pull request merged**